packages/api/src/schemas/envConfig.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://github.com/OfficeDev/TeamsFx/schemas/v1.0/envConfig.json",
+  "type": "object",
+  "description": "The schema of TeamsFx configuration.",
+  "properties": {
+    "$schema": {
+      "type": "string"
+    },
+    "azure": {
+      "type": "object",
+      "description": "The Azure resource related configuration.",
+      "properties": {
+        "subscriptionId": {
+          "type": "string",
+          "description": "The default subscription to provision Azure resources."
+        },
+        "resourceGroupName": {
+          "type": "string",
+          "description": "The default resource group of Azure resources."
+        },
+        "region": {
```

Comment:
Should we be consistent with Azure's wording? 
![image](https://user-images.githubusercontent.com/9698542/130568577-ff29208d-a333-4e27-b5b1-94f50f8ad4f6.png)

Reply:
updated to use `location` for resource group to keep consistent with the ARM template schema.